src/main/resources/application.properties.EXAMPLE

```diff
@@ -422,4 +422,7 @@ server.max-http-request-header-size=16384
 # Development Configuration
 spring.devtools.restart.enabled=false
 
-# EOL - Do not delete the following lines
+## Study Tag functionality
+#enable_study_tags=true|false
+
+# EOL - Do not delete the following lines
```

Comment:
I think you might have accidentally deleted the line after this. If you add it back then the builds should work.

Reply:
Done!